lmfdb/higher_genus_w_automorphisms/main.py

```diff
@@ -879,6 +949,65 @@ def render_passport(args):
                                credit=credit)
 
 
+#Generate topological webpage
+@higher_genus_w_automorphisms_page.route("/<fam>/<cc>") 
+def topological_action(fam, cc):
+    br_g, br_gp, br_sign = split_family_label(fam)
+    cc_list = cc_to_list(cc)
+    representative = fam + '.' + cc[2:]
+
+     #Get the equivalence class
+    topo_class = list(db.hgcwa_passports.search({'label': fam, 'topological': cc_list}))
+
+    GG = ast.literal_eval(topo_class[0]['group'])
+    gn = GG[0]
+    gt = GG[1]
+
+    gp_string=str(gn) + '.' + str(gt)
+    pretty_group=sg_pretty(gp_string)
+
+    bread_sign = label_to_breadcrumbs(br_sign)
+    bread_gp = label_to_breadcrumbs(br_gp)
+
+    bread = get_bread([(br_g, '../?genus='+br_g),('$'+pretty_group+'$','../?genus='+br_g + '&group='+bread_gp),
+                            (bread_sign, '../'+fam),
+                            ('Topological Orbit for ' + str(cc_list[0]) + ', ' + str(cc_list[1]), ' ') ])
+
+    title = 'One Orbit Under Topological Action'
+
+    downloads = [('Download Magma code', url_for(".hgcwa_code_download",  label=representative, download_type='rep_magma')),
+                      ('Download Gap code', url_for(".hgcwa_code_download", label=representative, download_type='rep_gap'))]
+
+    Lbraid={}
+
+    for element in topo_class:
+       if str(element['braid']) in Lbraid:
+           Lbraid[str(element['braid'])].append([element['passport_label'],
+                                              element['total_label'],
+                                              ' '])
```

Comment:
Why is the third entry a space here?  Is it just not used?

Reply:
Ah yes.  That was a workaround I came up with for a small display issue I was having.

Consider this page:
http://localhost:37777/HigherGenus/C/Aut/3.8-3.0.2-2-4-4/T.1.1

There are times when one topological equivalence class has multiple inequivalent braid equivalence classes in it, and those braid classes may have several different generating vectors in them. But braids classes preserve conjugacy classes, so the different generating vectors will have the same conjugacy classes. On the page above, without the space, it was printing the "3,3,5,5" and "4,4,5,5" twice, which looked silly.  

I tried messing around on the html side to have it only printed once, but I couldn't find a good solution.  So instead I'm passing in a blank space to print.  

If there is a better way to do this, I will happily change it!

Reply:
Here's maybe a better example:
http://localhost:37777/HigherGenus/C/Aut/7.24-14.0.2-2-2-2-2/T.1.1